src/mercury_engine_data_structures/formats/blsnd.py

```diff
@@ -16,16 +16,18 @@
 
 BLSND = Struct(
     "_magic" / Const(b"LSND"),
-    "version" / IfThenElse(
-        current_game_at_most(Game.SAMUS_RETURNS),
-        Const(0x000B0001, Hex(Int32ul)),
-        Const(0x000C0001, Hex(Int32ul))
+    "version"
+    / IfThenElse(
```

Comment:
Ruff makes these look rather ugly :(

Reply:
🤷

Reply:
Is there a way to tell ruff to exempt these? or is the only fix to just change all of those to use `=` syntax instead?

Reply:
I'm kinda fine with how it looks. Not the best, but it's still usable to not bother trying to make it look different.

Reply:
no this is really bad imo

Reply:
I'm strong on needing some sort of auto-formatting though. Especially to keep all the project settings in sync across all repos in the organisation.

Reply:
I unsucessfully tried to find some sort of rule for that. Is there anyway to do it without disabling ruff for each of those lines?

Reply:
black and ruff's formatter don't offer configuration to how the code is formatted. Best I can think of is wrapping the construct definitions in a "don't format this block"?